src/green/SphericalDiffuse.cpp

```diff
@@ -416,7 +403,18 @@ template class SphericalDiffuse<OneLayerTanh>;
 using dielectric_profile::OneLayerErf;
 template class SphericalDiffuse<OneLayerErf>;
 
-using dielectric_profile::MembraneTanh;
-template class SphericalDiffuse<MembraneTanh>;
+using dielectric_profile::OneLayerLog;
+template class SphericalDiffuse<OneLayerLog>;
+
+/*
+  FIXME: the numerical integration in logarithmic scale requires a step
```

Comment:
@ilfreddy @arnfinn Mentions of the `MembraneTanh` class should be removed from the code. This `FIXME` should become an issue, so we can keep track of it.